api/src/main/java/org/pmiops/workbench/config/RetryConfig.java

```diff
@@ -62,8 +64,20 @@ protected void logGivingUp(int retryCount, Throwable t) {
           String.format("%s unavailable, giving up after %d attempts", serviceName, retryCount), t);
     }
 
-    protected void logNoRetry(Throwable t) {
-      logger.log(Level.SEVERE, String.format("Exception calling %s", serviceName), t);
+    protected Level getLogLevel(int responseCode) {
+      switch (responseCode) {
+        case HttpServletResponse.SC_NOT_FOUND:
```

Comment:
I think there are some other ones that we want to log here? BadRequest seems like one we should potentially log as only info or warning?

Reply:
Oops, already merged. :)

I intentionally didn't include bad request -- we generally should not be doing anything that produces bad requests to other systems. Which may in some cases mean mirroring some of their validation. If that proves to be a problem in certain cases, we can alter the logging in those cases specifically.